src/components/Map.vue

```diff
@@ -1196,16 +1216,22 @@ export default {
         }
       }
     },
+    getFwwLayer() {
+      this.fwwLayer = esri.dynamicMapLayer({
+        url: "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Forecasts_Guidance_Warnings/watch_warn_adv/MapServer",
+        layers: [0, 1],
+        format: "image/png",
```

Comment:
```suggestion
        f: "image/png",
```
looking at their docs https://esri.github.io/esri-leaflet/api-reference/layers/dynamic-map-layer.html I think we want 'format' to be 'f', should help with layer not loading